lib/elixir/lib/system.ex

```diff
@@ -880,6 +880,52 @@ defmodule System do
     :init.stop(String.to_charlist(status))
   end
 
+  @doc ~S"""
+  Executes the given `command` in the current OS shell.
```

Comment:
```suggestion
  Executes the given `command` in the OS shell.
```

"current" makes me think that it will read the "SHELL" env var or something similar